modules/nf-neuro/preproc/topup/main.nf

```diff
@@ -3,12 +3,12 @@ process PREPROC_TOPUP {
     label 'process_single'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        "https://scil.usherbrooke.ca/containers/scilus_2.0.2.sif":
-        "scilus/scilus:2.0.2"}"
+        'https://scil.usherbrooke.ca/containers/scilus_2.0.2.sif':
+        'scilus/scilus:2.0.2' }"
 
     input:
         tuple val(meta), path(dwi), path(bval), path(bvec), path(b0), path(rev_dwi), path(rev_bval), path(rev_bvec), path(rev_b0)
-        val(config_topup)
```

Comment:
To be semantically correct, here you should use a [value channel](https://www.nextflow.io/docs/latest/process.html#process-multiple-input-channels) (just learned about that) instead of `each`. The problem with `each` comes if a user provides **more than one** config in the `config_topup` channel. Then, we would have multiple Topup results per subject, which is bad. The `value channel` in comparison will just repeat its singular value for ever.